### PR TITLE
Update setup-konnectivity.md

### PR DESCRIPTION
**Pull request by daixiang0**
Add missing volume config.
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

### CODE REVIEW

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
Comment (cheftako) [inline content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:40]: volumes rather than voluimens?
Also this step is specifically (only) needed for UDS connections. We should indicate that in the note.

Comment (daixiang0) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:40]: Now the whole demo uses UDS.

Comment (cheftako) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md]: Agreed as its easier to maintain a single path for demonstration. However it is not the only supported option. Their is for instance an option to use mTLS & GRPC. That having been said you seem to have added the indicator, thank you :D

Comment (cheftako) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md]: /lgtm

Comment (daixiang0) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md]: Done.

**Comment by cheftako** [comment]
@daixiang0 I believe you need to get an approval from one of the english content owners at https://github.com/kubernetes/website/blob/7716cd69d4c7058caa60e5b8edda220b39e34d0c/OWNERS_ALIASES#L24

**Comment by daixiang0** [comment]
@sig-docs-en-owners

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
Comment (tengqm) [inline content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:48]: This indentation is weird. please double check.

Comment (daixiang0) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:48]: Done.

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
Comment (tengqm) [inline content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:42]: This still looks incomplete and confusing..

**Comment by tengqm** [comment]
/hold
The sample YAML snippet has to be fixed.

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
Comment (daixiang0) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:42]: It works at local.

Comment (tengqm) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:42]: Yes. It works on your machine. But when you are writing it as a document, you will assume the reader has no proper context. It's like copying some incomplete complete code snippet from C program and sharing it with others. Hey, this works on my machine:

```
     printf("%d", result);
   }
   check_result();
```

The first line in my example has no context. You have no idea if it is from a conditional block or a loop.
Now check your YAML.

Comment (daixiang0) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:44]: Done.

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
Comment (onlydole) [inline content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:41]:
```suggestion
```yaml
```

**Comment by daixiang0** [comment]
ping @tengqm @onlydole

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
Comment (tengqm) [inline content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:53]: pls indent line 41-53 by three spaces so the block aligns to the first "I" character on line 40.

Comment (daixiang0) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:53]: Done.

**Comment by tengqm** [comment]
/approve
Thanks.

**Review by onlydole — APPROVED** [review]
Thank you, @daixiang0!

/lgtm
/hold cancel

**Pull request merged**
